mobile/androidMediaSessionAPI.html

```diff
@@ -0,0 +1,232 @@
+<!DOCTYPE html>
+<!--
+*
+* Copyright (C) 2015, bitmovin GmbH, All Rights Reserved
```

Comment:
that copyright notice seems a bit outdated

Reply:
should we just remove it? There are other samples where there is no copyright notice

Reply:
Just noticed we don't have a `LICENSE` file in this repo, will add one. You can drop the header, please do this then for the other samples as well (in another PR)